CONTRIBUTING.md

```diff
@@ -49,18 +67,39 @@ The following will create a universal [wheel](https://pythonwheels.com/) ready
 to be installed in any python 2 or python 3 environment.
```

Comment:
We should revisit this, since it's not clear what packaging fits in for someone who is doing development. The easiest workflow for development is to simply `cd rsconnect-jupyter` and execute `pip install -e ./` so that changes are reflected in real time.

Reply:
Agreed. I'll make a second pass and work on cleaning things up in a separate pull request. Feel free to leave notes here; I'm making a list.